pinot-filesystem/src/main/java/com/linkedin/pinot/filesystem/LocalPinotFS.java

```diff
@@ -40,6 +40,12 @@ public LocalPinotFS() {
   public void init(Configuration configuration) {
   }
 
+  @Override
+  public boolean mkdir(URI uri) throws IOException {
+    FileUtils.forceMkdir(new File(uri));
+    return true;
```

Comment:
Always returning true is not useful.

Reply:
agree, that's why i made the interface defn void. For now, will leave as boolean with the rest as you mentioned in an earlier comment for consistency. Will clean all of them up together after doing some more careful investigation in our code.